client_test.go

```diff
@@ -321,12 +320,10 @@ func CreatePermission(t *testing.T, client *gokeycloak.GoKeycloak, idOfClient st
 	return tearDown, *createdPermission.ID
 }
 
-func CreateClient(t *testing.T, client *gokeycloak.GoKeycloak, newClient *gokeycloak.Client) (func(), string) {
+func CreateClient(t *testing.T, client *gokeycloak.GoKeycloak, newClient *gokeycloak.CreateClientRequest) (func(), string) {
 	if newClient == nil {
-		newClient = &gokeycloak.Client{
-			ClientID: GetRandomNameP("ClientID"),
-			Name:     GetRandomNameP("Name"),
-			BaseURL:  gokeycloak.StringP("http://example.com"),
+		newClient = &gokeycloak.CreateClientRequest{
+			RedirectUris: []string{"http://127.0.0.1/"},
```

Comment:
Why are we hardcoding this? can it be an invalid URL such as any string like "invalidurl"?